provisioning/miniprov/hare_mp/main.py

```diff
@@ -223,6 +223,36 @@ def init(args):
         raise RuntimeError(f'Error while initializing cluster :key={error}')
 
 
+def start(args):
+    try:
+        path = os.getenv('PATH')
+        if path:
```

Comment:
If path is None, what will happen then?

Reply:
Good point, fix by setting a default value to path if PATH is None. Removed the check.